ACMAS/app/ACMAS_Web/templates/upload-OCR.html

```diff
@@ -105,7 +105,7 @@
             </a>
         </div>
         <span class="upload-title-text">ACMAS Upload</span>
-        <form id="school-course-input" method="POST" enctype="multipart/form-data" action="{% url 'uploadOCR' %}"> {% csrf_token %}
+        <form id="school-course-input" method="POST" enctype="multipart/form-data" action="{% url 'uploadOCR' %}">
```

Comment:
Very nice we don't need it anymore!

Reply:
We still need it in the future, but it was causing issues presently. See my comment to Susan in the PR on the same topic.